Dockerfile

```diff
@@ -20,4 +20,16 @@ EXPOSE 80
 
 WORKDIR /cms
 RUN /cms/create-users
```

Comment:
sorry, my bad, didn't checked the full file content. 
Anyway sometimes i have an issues with djangocms-installer and sqlite - after first install i get syncdb and migrations faked and create users via prompt I have to do it again manually (syncdb and create a user, like it wasn't done before).
So if you encounter such issue, i think it would be enough to move this at the bottom, after all syncdb's and migrations as a workaround.